### PR TITLE
AAC quality improvements

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: contributing-to/reference-architectures.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 ## What is a Reference Architecture?
 
-The term *reference architecture* means a lot of things, but in the context of the Azure Architecture Center, it refers to a specific type of guidance artifact, found [here](https://docs.microsoft.com/azure/architecture/reference-architectures/). 
+The term *reference architecture* means a lot of things, but in the context of the Azure Architecture Center, it refers to a specific type of guidance artifact, found [here](/azure/architecture/reference-architectures/).
```
Comment (MikeWasson) [inline contributing-to/reference-architectures.md]: This file doesn't get published so we need the full URL here

File: contributing-to/reference-architectures.md [hunk 2]
```diff
@@ -28,20 +28,20 @@ In other cases, the series contains a set of recommended alternatives for a sing
 
 - **Concrete**. Our reference architectures show specific Azure services, as opposed to logical components such as "data store" or "stream processing."
 
-  > We know that more generalized architectures, showing logical components, are also very useful. However, we treat these as a separate type of guidance artifact. For example, see [here](https://docs.microsoft.com/azure/architecture/guide/architecture-styles/web-queue-worker) and [here](https://docs.microsoft.com/azure/architecture/data-guide/concepts/big-data).
+  > We know that more generalized architectures, showing logical components, are also very useful. However, we treat these as a separate type of guidance artifact. For example, see [here](/azure/architecture/guide/architecture-styles/web-queue-worker) and [here](/azure/architecture/data-guide/concepts/big-data).
```
Comment (MikeWasson) [inline contributing-to/reference-architectures.md]: same comment

File: contributing-to/reference-architectures.md [hunk 3]
```diff
@@ -114,7 +114,7 @@ These are four sections (H2) that describe non-functional characteristics of the
 - Manageability
 - Security
 
-These correspond to 4 of our 5 [pillars of software quality](https://docs.microsoft.com/azure/architecture/guide/pillars).
+These correspond to 4 of our 5 [pillars of software quality](/azure/architecture/guide/pillars).
```
Comment (MikeWasson) [inline contributing-to/reference-architectures.md]: same comment

File: docs/antipatterns/chatty-io/index.md [hunk 4]
```diff
@@ -12,7 +12,7 @@ ms.custom: seodec18
 
 # Chatty I/O antipattern
 
-The cumulative effect of a large number of I/O requests can have a significant impact on performance and responsiveness.
+The cumulative effect of a large number of I/O requests can have a significant effect on performance and responsiveness.
```
Comment (MikeWasson) [inline docs/antipatterns/chatty-io/index.md]: I think 'impact' is OK here. Effect is any result of a cause, impact implies a negative effect which is the case here. (We should avoid 'impact' as a verb however)

File: docs/aws-professional/index.md [hunk 5]
```diff
@@ -107,7 +107,7 @@ The syntax and structure of these interfaces are different from their AWS equiva
 
 ## Regions and zones (high availability)
 
-Failures can vary in the scope of their impact. Some hardware failures, such as a failed disk, may affect a single host machine. A failed network switch could affect a whole server rack. Less common are failures that disrupt a whole data center, such as loss of power in a data center. Rarely, an entire region could become unavailable.
+Failures can vary in the scope of their effect. Some hardware failures, such as a failed disk, may affect a single host machine. A failed network switch could affect a whole server rack. Less common are failures that disrupt a whole data center, such as loss of power in a data center. Rarely, an entire region could become unavailable.
```
Comment (MikeWasson) [inline docs/aws-professional/index.md]: 'Impact' is OK here

File: docs/aws-professional/services.md [hunk 6]
```diff
@@ -51,12 +51,12 @@ For an overview of Azure for AWS users, see [Introduction to Azure for AWS profe
...
+|               Backup               |                                     None                                      |                                                                                                                   [Azure Backup](https://azure.microsoft.com/services/backup/)                                                                                                                   | Backup and archival solutions allow files and folders to be backed up and recovered from the cloud, and provide offsite protection against data loss. There are two components of backup &mdash; the software service that orchestrates backup/retrieval and the underlying backup storage infrastructure. |
```
Comment (MikeWasson) [inline docs/aws-professional/services.md]: I think a recent PR creates a conflict here - we added 'AWS Backup'

File: docs/best-practices/background-jobs.md [hunk 7]
```diff
@@ -62,7 +62,7 @@ If you use a schedule-driven task that must run as a single instance, be aware o
 
 ## Returning results
 
-Background jobs execute asynchronously in a separate process, or even in a separate location, from the UI or the process that invoked the background task. Ideally, background tasks are “fire and forget” operations, and their execution progress has no impact on the UI or the calling process. This means that the calling process does not wait for completion of the tasks. Therefore, it cannot automatically detect when the task ends.
+Background jobs execute asynchronously in a separate process, or even in a separate location, from the UI or the process that invoked the background task. Ideally, background tasks are “fire and forget” operations, and their execution progress has no effect on the UI or the calling process. This means that the calling process does not wait for completion of the tasks. Therefore, it cannot automatically detect when the task ends.
```
Comment (MikeWasson) [inline docs/best-practices/background-jobs.md]: I think 'impact' is OK here (implies negative effect)

File: docs/example-scenario/index.md [hunk 8]
```diff
@@ -55,7 +57,7 @@ Jump to: [AI scenarios](#ai-scenarios) | [Application scenarios](#application-sc
                     <div class="cardText">
                         <a href="https://azure-architecture.uservoice.com/forums/918625-architecture-guidance" data-linktype="external">
                             <div class="cardSize cardsF">
-                                <div class="cardPadding">
+                                <!-- <div class="cardPadding"> -->
```
Comment (MikeWasson) [inline docs/example-scenario/index.md]: Is this div not needed?

File: contributing-to/reference-architectures.md (hunk 1 above)
Comment (alexbuckgit) [reply contributing-to/reference-architectures.md]: Ah, that makes sense. Fixed.

File: contributing-to/reference-architectures.md (hunk 2 above)
Comment (alexbuckgit) [reply contributing-to/reference-architectures.md]: Fixed

File: contributing-to/reference-architectures.md (hunk 3 above)
Comment (alexbuckgit) [reply contributing-to/reference-architectures.md]: Fixed

File: docs/antipatterns/chatty-io/index.md (hunk 4 above)
Comment (alexbuckgit) [reply docs/antipatterns/chatty-io/index.md]: Reverted

File: docs/aws-professional/index.md (hunk 5 above)
Comment (alexbuckgit) [reply docs/aws-professional/index.md]: Reverted

File: docs/example-scenario/index.md (hunk 8 above)
Comment (alexbuckgit) [reply docs/example-scenario/index.md]: Must've hit the comment shortcut key by mistake. Reverted.

File: docs/best-practices/background-jobs.md (hunk 7 above)
Comment (alexbuckgit) [reply docs/best-practices/background-jobs.md]: Reverted.

File: docs/aws-professional/services.md (hunk 6 above)
Comment (alexbuckgit) [reply docs/aws-professional/services.md]: Reverted to version in master.

**Pull request merged**